docs/courses/creating-courses.md

```diff
@@ -0,0 +1,19 @@
+## Creating a new LibreLingo course
+
+Beginners should not be bothered by this page. This page is for those who want to create new courses from scratch, which is for those who have sufficient understanding of the project and how the courses work. Please continue at your own discretion.
```

Comment:
> Beginners should not be bothered by this page.

Ideally beginners would edit some other course before creating a new one, but that will not always be the case. Also nobody should be bothered because all aspects of this documentation are fun and exciting to read :stuck_out_tongue_closed_eyes: 

I would maybe rather say something like `Before creating a new course, we recommend you try contributing to an existing one or at least exploring other parts of this documentation.`

Reply:
> Please continue at your own discretion.

That's a little strong, I would use this phrase rather if there's some physical danger ahead.

Reply:
I would probably replace the overall paragraph with this:

> Before creating a new course, we recommend you try contributing to an existing one or at least exploring other parts of this documentation. In order to be successful in creating a new course from scratch, you'll need sufficient understanding of the project and how the courses work.